examples/keystone-in-nextjs/README.md

```diff
@@ -0,0 +1,44 @@
+# Keystone in Next.js
+
+Keystone could double down as a powerful data engine within Next.js applications without having to host a separate Keystone server. This is made possible by Keystone's `getContext` API.
+
+- **CRUD data within your Next.js server**: You can use the Keystone data APIs directly in Next.js `getStaticProps` or `getServerSideProps` to CRUD data. ⚡️
+- **CRUD data from browser**: You can use the generated Keystone GraphQL schema to setup your own GraphQL server in a Next.js route. This enables you to send GraphQL requests from the browser. 🤯 (refer to [/pages/api/graphql-yoga.ts](/pages/api/graphql-yoga.ts) for implementation details)
```

Comment:
```suggestion
- **CRUD data from browser**: You can use the generated Keystone GraphQL schema to setup your own GraphQL server in a Next.js route. This enables you to send GraphQL requests from the browser. 🤯 (refer to [/pages/api/graphql.ts](/pages/api/graphql.ts) for implementation details)
```

Reply:
Yup. I have some changes in local. Will push this along with that.